client/src/theme/variables/width.scss

```diff
@@ -7,4 +7,7 @@
   --parsec-min-content-width: 35rem;
   --parsec-max-content-width: 78rem;
   --parsec-max-title-width: 20rem;
+  --parsec-sidebar-menu-width: 300px;
+  --parsec-sidebar-menu-min-width: 0px;
+  --parsec-sidebar-menu-max-width: 500px;
```

Comment:
Might we use relatives size?

Reply:
we could, but it would complicate a lot the other places you commented, I'm not against pixels for something as primary as the main menu.

Reply:
will be managed in https://github.com/Scille/parsec-cloud/issues/5945